google/cloud/forseti/common/gcp_api/admin_directory.py

```diff
@@ -149,33 +149,30 @@ def __init__(self, **kwargs):
 class AdminDirectoryClient(object):
     """GSuite Admin Directory API Client."""
 
-    DEFAULT_QUOTA_PERIOD = 100.0
-
     def __init__(self, global_configs, **kwargs):
         """Initialize.
 
         Args:
             global_configs (dict): Global configurations.
             **kwargs (dict): The kwargs.
         """
-        max_calls = global_configs.get('max_admin_api_calls_per_100_seconds')
-        quota_period = self.DEFAULT_QUOTA_PERIOD
-        if not max_calls:
-            max_calls = global_configs.get('max_admin_api_calls_per_day')
-            quota_period = 86400.0
-            LOGGER.error('Configuration is using a deprecated directive: '
-                         '"max_admin_api_calls_per_day". Please switch to '
-                         'using "max_admin_api_calls_per_100_seconds" instead. '
-                         'See the sample configuration file for reference.')
-
         credentials = api_helpers.get_delegated_credential(
             global_configs.get('domain_super_admin_email'),
             REQUIRED_SCOPES)
-        self.repository = AdminDirectoryRepositoryClient(
-            credentials=credentials,
-            quota_max_calls=max_calls,
-            quota_period=quota_period,
-            use_rate_limiter=kwargs.get('use_rate_limiter', True))
+
+        admin_api_config = global_configs.get('admin')
```

Comment:
Since this same pattern is used in every API client, I recommend a bit of a refactor to centralize the extraction of the quota config.

in api_helpers:

```
def get_ratelimiter_config(global_configs, api_name):
  max_calls = global_configs.get(api_name, {}).get('max_calls')
  quota_period = global_configs.get(api_name, {}).get('period')
  return (max_calls, quota_period)
```

In each API client:

```
(max_calls, quota_period) = api_helpers.get_ratelimiter_config(global_configs, self.API_NAME)
self.repository = Client(
  quota_max_calls=max_calls,
  quota_period=quota_period,
   use_rate_limiter=kwargs.get('use_rate_limiter', True))
```

Since every client will disable the ratelimiter if max_calls is set to None (which it will be by the get if there is no data) then you don't need the if, else block.